src/context/TokenContext.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {
```

Comment:
The only purpose of this context is to pass the selected token address to the new action form, right? If so, can that not be achieved using existing `dispatchGlobalEvent` solution, similarly to how we set the action type? It's one less moving part to worry about